Userland/Utilities/unzip.cpp

```diff
@@ -140,6 +143,28 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         TRY(Core::System::chdir(output_directory_path));
     }
 
+    if (list_files) {
+        outln("  Length     Date      Time     Name");
+        outln("--------- ---------- --------   ----");
+        u16 members_count = 0;
```

Comment:
If I've understood correctly, the original ZIP format had a limit of 65535 entries per archive, but that limit was raised in some subsequent addendum to the ZIP format spec.

In other words, should we not use a larger data type here? :)